examples/lifecycle/client.spec.ts

```diff
@@ -57,7 +58,7 @@ describe('lifecycle typed client', () => {
   })
 
   test('deploy_bare_opt_in', async () => {
-    const createResult = await client.deploy({ createArgs: { onCompleteAction: 'opt_in' } })
+    const createResult = await client.deploy({ createArgs: LifeCycleAppCallFactory.create.bare({ onCompleteAction: 'opt_in' }) })
```

Comment:
`create/update/deleteArgs` can now be set with the result of a call factory.

Reply:
Is that intuitive to use?